projects/hipblaslt/CMakeLists.txt

```
@@ -1,490 +1,570 @@
-  find_program(LLVM_COV_EXECUTABLE llvm-cov HINTS ${LLVM_BIN_DIR} REQUIRED)
-
-  # Use LLVM libraries and headers
-  find_package(LLVM REQUIRED CONFIG)
-
-  # Modern: Use target_* for compiler and linker flags
-  target_compile_options(hipblaslt PRIVATE
-    -fprofile-instr-generate
-    -fcoverage-mapping
-    -fvisibility=default
-  )
-
-  target_link_options(hipblaslt PUBLIC
-    -fprofile-instr-generate
-  )
-
-  target_include_directories(hipblaslt PRIVATE
-    ${LLVM_INCLUDE_DIRS}
-  )
-
-  target_link_directories(hipblaslt PRIVATE
-    ${LLVM_LIBRARY_DIRS}
-  )
-
-  # Print status message
-  message(STATUS "LLVM found at: ${LLVM_BIN_DIR}")
-  message(STATUS "llvm-profdata: ${LLVM_PROFDATA_EXECUTABLE}")
-  message(STATUS "llvm-cov: ${LLVM_COV_EXECUTABLE}")
-
-  add_custom_target(coverage
-    COMMAND ${CMAKE_COMMAND} -E env
-            LLVM_PROFILE_VERBOSE=1
-            LLVM_PROFILE_FILE=./hipblaslt-test_%m.profraw
-            ./clients/staging/hipblaslt-test
-            --output-on-failure
-    COMMAND ${CMAKE_COMMAND} -E env
-            LLVM_PROFILE_VERBOSE=1
-            LLVM_PROFILE_FILE=./hipblaslt-coverage_%m.profraw
-            ./clients/staging/sample_hipblaslt_ext_op_amax
-            --output-on-failure
-    COMMAND ${CMAKE_COMMAND} -E env
-            LLVM_PROFILE_VERBOSE=1
-            LLVM_PROFILE_FILE=./hipblaslt-coverage_%m.profraw
-            ./clients/staging/sample_hipblaslt_ext_op_layernorm
-            --output-on-failure
-    COMMAND ${CMAKE_COMMAND} -E env
-            LLVM_PROFILE_VERBOSE=1
-            LLVM_PROFILE_FILE=./hipblaslt-coverage_%m.profraw
-            ./clients/staging/sample_hipblaslt_weight_swizzle_padding
-            --output-on-failure
-    COMMAND ${CMAKE_COMMAND} -E env /bin/sh -c "${LLVM_PROFDATA_EXECUTABLE} merge -sparse ${CMAKE_BINARY_DIR}/*.profraw -o ${CMAKE_BINARY_DIR}/coverage_test.profdata"
-
-    COMMAND ${LLVM_COV_EXECUTABLE}
-            report
-            -object ./library/libhipblaslt-d.so.${hipblaslt_SOVERSION}
-            -instr-profile=./coverage_test.profdata
-            --ignore-filename-regex=.*/build/debug/.*
-    COMMAND ${LLVM_COV_EXECUTABLE}
-            show
-            -object ./library/libhipblaslt-d.so.${hipblaslt_SOVERSION}
-            -instr-profile=./coverage_test.profdata
-            -format=html -output-dir=coverage-report
-            --ignore-filename-regex=.*/build/debug/.*
-
-    COMMAND tox -c ../../tensilelite -v --workdir /tmp/.tensile-tox -e py3 -- ../../tensilelite/Tensile/Tests -m common --cov=../../tensilelite --cov-report=html
-    WORKING_DIRECTORY ${CMAKE_BINARY_DIR}
-
-    VERBATIM
-  )
-
-  add_dependencies(coverage hipblaslt-test
-                            sample_hipblaslt_ext_op_amax
-                            sample_hipblaslt_ext_op_layernorm
-                            sample_hipblaslt_weight_swizzle_padding)
+if(ROCM_LIBRARIES_SUPERBUILD OR NOT IS_SUBPROJECT)
```

```suggestion
if(ROCM_LIBRARIES_SUPERBUILD OR NOT HIPBLASLT_IS_SUBPROJECT)
```

(also see related comment where this variable is set)

---

Fixed